### DISCUSSION TITLE
Nomination thread for next Steering Committee

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#81] [Steering Committee]
To make self-nomination more convenient for everyone, I've created this post. To nominate yourself, simply reply using the following format:

"I would like to nominate myself for the election of the next steering committee."

Optionally, you can also share more about yourself beyond this nomination.

**Comment by passionnn** [comment]
Hey guys, I would like to nominate myself for the election of the next steering committee.

I am passionate about crypto and have a strong background in project research and community management. I have been an active contributor to our community and believe that my experience and dedication make me a suitable candidate for the steering committee.

**Comment by casinoyoda** [comment]
I would like to nominate myself for the election of the next steering committee.

I am the technical supporter and also a member of the steering committee currently. I am responsible for maintaining the infrastructure of DCDAO, such as Snapshot, Profiles, and some tools to help communication in the community. I also help manage the funds in the DCDAO treasury.

**Comment by nblogist** [comment]
I would like to nominate my self for the next steering committee
I am currently looking at any integration requirements of DCDAO, happy to liaison between external team.

**Comment by passionnn** [comment]
Hey PD,

I wrote u a message on tg, thx~

PartnershipDude ***@***.***> 于2024年5月8日周三 03:33写道：
